rules/namespace.go

```diff
@@ -292,10 +294,10 @@ func (nss *Namespaces) Namespace(name string) (*Namespace, error) {
 }
 
 // AddNamespace adds a new namespace to the namespaces structure and persists it
```

Comment:
Can you add a comment saying the boolean value returned indicating whether the namespace is revived or not?